src/style/app.module.css

```diff
@@ -1256,9 +1313,9 @@ hr {
   background-color: var(--white-color);
   margin: 0 4px;
   justify-content: space-between;
-  box-shadow: 0 3px 8px var(--black-shadow-color);
   align-items: center;
-  border-radius: 20px;
+  border: 1px solid var(--input-shadow);
+  border-radius: 8px;
 }
 .ctacards span {
   color: var(--dimp-white);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`